src/receivers/ExpressReceiver.ts

```diff
@@ -41,6 +41,7 @@ interface InstallerOptions {
   authVersion?: InstallProviderOptions['authVersion']; // default 'v2'
   metadata?: InstallURLOptions['metadata'];
   installPath?: string;
+  directInstallUrlEnabled?: boolean; // see https://api.slack.com/start/distributing/directory#direct_install
```

Comment:
Do we prefer something simpler like `directInstall: boolean`?

Reply:
I like the shorter name but that's only my personal preference

Reply:
Got two votes 😄 Renamed.